client/src/App.jsx

```diff
@@ -12,7 +16,7 @@ const Accordian = styled.div`
 `;
 
 const Title = styled.h3`
-  border-top: 1px solid gray;
+  border-top: 1px solid rgb(205, 205, 177);
   background: whitesmoke;
   padding: 20px 0 20px 25px;
   cursor: pointer;
```

Comment:
can this go into your css file?

Reply:
it could but it's also a react styled-component thing so idk what's best practice is actually